warp/src/main/java/com/schibsted/nmp/warp/components/WarpModal.kt

```diff
@@ -0,0 +1,142 @@
+package com.schibsted.nmp.warp.components
+
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.rememberScrollState
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.foundation.verticalScroll
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.Close
+import androidx.compose.material3.Card
+import androidx.compose.material3.CardDefaults
+import androidx.compose.material3.Icon
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.window.Dialog
+import androidx.compose.ui.window.DialogProperties
+import com.schibsted.nmp.warp.theme.WarpTheme.colors
+import com.schibsted.nmp.warp.theme.WarpTheme.dimensions
+
+@Composable
+fun WarpModal(
+    title: String,
+    body: String,
+    onDismiss: () -> Unit,
```

Comment:
the order of the params could use a bit of love

Reply:
Ive put the mandatory ones on top, the the optional ones

Reply:
I see, what I meant was that the related ones could be grouped together:
confirmButtonText & onConfirm, dismissButtonText & onDismiss
also iirc, modifier should always be the first param of a Composable